docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -440,47 +480,48 @@ workload networks
 ```
 
 </details>
+</TabItem>
+<TabItem value="2" label="GCP permissions">
+<details><summary>
+Show permissions to be granted to Aiven
+</summary>
+- `get` on the GCP project to retrieve project-level quotas
+
+- `list` all the regions in the project
+
+- `get` on each region to retrieve region-level quotas
+
+- `ListTimeSeriesRequest` using the Google `monitoring_v3.MetricServiceClient` for the
+  following metrics:
+
+  - `compute.googleapis.com/quota/static_routes_per_vpc_network/usage`
+  - `compute.googleapis.com/quota/static_routes_per_vpc_network/limit`
+
+</details>
```

Comment:
I wonder where these details come from?

I would say that the customer service account that will run the TF script in the Customer Google project needs the permissions listed in the following Google [predefined roles](https://cloud.google.com/iam/docs/understanding-roles):
- roles/iam.serviceAccountAdmin (setup impersonation to Privilege-Bearing Service Account)
- roles/resourcemanager.projectIamAdmin (grant use of Customer project to P-B SA)
- roles/compute.instanceAdmin.v1 (manage networks and instances)
- roles/compute.securityAdmin (create firewall rules)
- enable [IAM API](https://console.developers.google.com/apis/api/iam.googleapis.com) to create P-B SA
- enable [CRM API](https://console.cloud.google.com/apis/library/cloudresourcemanager.googleapis.com) to set IAM policies to P-B SA.